testsuite/kubernetes/deployment.py

```diff
@@ -187,6 +190,13 @@ def add_volume(self, volume: Volume):
         mounts = self.template.setdefault("volumes", [])
         mounts.append(asdict(volume))
 
+    def restart(self):
```

Comment:
Isnt this equivalent to `rollout(hard=True)`?

Reply:
I need graceful termination, unlike the hard rollout solution. Plus, I thought hard rollout is there only until limitador pod rollout bug is fixed